src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -473,14 +477,15 @@
   background-color: var(--#{$menu-toggle}__button--BackgroundColor);
   border: 0;
 
-  &:has(.#{$menu-toggle}__controls) {
-    padding-inline-start: 0;
-  }
-
   &.pf-m-text {
     display: inline-flex;
     align-items: baseline;
-    padding-inline-start: var(--#{$menu-toggle}__button--m-text--PaddingInlineStart);
+  }
+
+  @at-root .#{$menu-toggle}.pf-m-split-button > .#{$check}.pf-m-standalone,
+  &:has(> .#{$menu-toggle}__controls:only-child) {
```

Comment:
Targets a split button item that has only a checkbox or icon and reduces the padding per design